### DISCUSSION TITLE
Tooltip calendar starts weeks with Sunday

### DISCUSSION THREAD

**Discussion by nauski** [Alexays/Waybar#2235] [Q&A] [answered]
Is it possible to change this to Monday?

**Comment by LukashonakV** [comment, marked as answer]
It depends on LC_TIME variable is configured in the system. This variable impacts any aspects in the system related to the time/date functionalities.
But you can try to use LC_TIME locally the only for waybar.
For example: `LC_TIME="en_GB.UTF-8" waybar`. Make sure en_GB.UTF-8 locale is installed in the system

**Reply by nauski** [reply]
Perfect, thank you for the quick reply!
